pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -333,3 +333,73 @@ class ClientConfigurationSerializer(serializers.Serializer):
     """Configuration settings for the ansible-galaxy client."""
 
     default_distribution_path = serializers.CharField(allow_null=True)
+
+
+class CollectionVersionSearchListSerializer(CollectionVersionListSerializer):
+    """Cross-repo search results."""
+
+    # All of these fields have to operate differently from the parent class
+    pulp_id = serializers.CharField(source="collectionversion_id")
+    repository_id = serializers.CharField()
+    repository_name = serializers.CharField(source="reponame")
+    namespace = serializers.CharField()
+    name = serializers.CharField()
+    namespace = serializers.CharField()
+    version = serializers.CharField()
+    created_at = serializers.CharField(source="cv_created_at")
+    updated_at = serializers.CharField(source="cv_updated_at")
+    requires_ansible = serializers.CharField(source="cv_requires_ansible")
+    dependencies = serializers.CharField(source="cv_dependencies")
+
+    # method fields
+    href = serializers.SerializerMethodField()
+    is_highest = serializers.SerializerMethodField()
+    signatures = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.CrossRepositoryCollectionVersionIndexView
+
+        fields = (
+            # "pk",
+            "pulp_id",
+            "repository_name",
+            "repository_id",
+            "namespace",
+            "name",
+            "version",
+            "created_at",
+            "updated_at",
+            "requires_ansible",
+            "dependencies",
+            "href",
+            "is_highest",
+            "is_deprecated",
+            "is_signed",
+            # "tags",
+            "signatures",
+        )
+
+    def get_collection_version(self, obj):
+        """Shortcut in case the collectionversion property moves around."""
+        return obj.collectionversion
+
+    def get_is_highest(self, obj):
```

Comment:
Can you remove get_is_highest and get_collection_version for now? It's slowing the API endpoint way down and isn't currently implemented yet.

This change speeds the response up by about 2 seconds